base/entries_test.go

```diff
@@ -0,0 +1,134 @@
+package base
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/qri-io/dataset"
+	"github.com/qri-io/dataset/dsio"
+)
+
+func TestReadEntriesArray(t *testing.T) {
```

Comment:
I've added some happy-path testing here. Knowing you're not a fan of table tests, I thought we'd try adding a "description" to the anon struct to help some of our issues with table tests.

Many of our table tests tend to take on this shape:
```go
cases := []struct{
}{
   // test this type of thing
  {},
  {},

  // test that type of thing
  {},
  {},
}
```
Not knowing _what a test case is checking_ is causing us problems. I think all new table tests should be required to include a description field that annotates what that case is testing, and we should use it to get around the problematic output of `case 15: ...`.

cc @ramfox

Reply:
I like using a description much better. It makes it possible to Ctrl+F for the failing test case, which `case 15` doesn't allow for.